go/tools/bazel_testing/BUILD.bazel

```diff
@@ -1,17 +1,22 @@
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("@go_host_compatible_sdk_label//:defs.bzl", "HOST_COMPATIBLE_SDK")
 load("//go:def.bzl", "go_library")
 
+_HOST_COMPATIBLE_SDK_ROOT = HOST_COMPATIBLE_SDK.same_package_label("ROOT")
```

Comment:
Is `_HOST_COMPATIBLE_SDK_ROOT` the same as `HOST_COMPATIBLE_SDK`?

Reply:
Yes, simplified